packages/terra-consumer-nav/src/components/nav-items/NavItem.scss

```diff
@@ -21,6 +21,15 @@
     }
   }
 
+  .nav-item-icon svg {
```

Comment:
this wont' suffice for font icons... will need to set a font-size to control those